{"stats": {"errors": 1, "failures": 0, "passes": 0, "skipped": 0, "encoding": "UTF-8", "total": 1}, "results": [{"classname": "nose.failure:Failure", "name": "runTest", "time": 0.00040340423583984375, "type": "error", "errtype": "builtins.ImportError", "message": "Install xlrd >= 0.9.0 for Excel support", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/site-packages/pandas/io/excel.py\", line 261, in __init__\n    import xlrd\nModuleNotFoundError: No module named 'xlrd'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/opt/conda/lib/python3.6/site-packages/nose/failure.py\", line 39, in runTest\n    raise self.exc_val.with_traceback(self.tb)\n  File \"/opt/conda/lib/python3.6/site-packages/nose/loader.py\", line 417, in loadTestsFromName\n    addr.filename, addr.module)\n  File \"/opt/conda/lib/python3.6/site-packages/nose/importer.py\", line 47, in importFromPath\n    return self.importFromDir(dir_path, fqname)\n  File \"/opt/conda/lib/python3.6/site-packages/nose/importer.py\", line 94, in importFromDir\n    mod = load_module(part_fqname, fh, filename, desc)\n  File \"/opt/conda/lib/python3.6/imp.py\", line 235, in load_module\n    return load_source(name, filename, file)\n  File \"/opt/conda/lib/python3.6/imp.py\", line 172, in load_source\n    module = _load(spec)\n  File \"<frozen importlib._bootstrap>\", line 684, in _load\n  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n  File \"/home/jeetds1994/python-for-loops-lab-data-science-alpha/test/index_test.py\", line 2, in <module>\n    from ipynb.fs.full.index import *\n  File \"/home/jeetds1994/python-for-loops-lab-data-science-alpha/index.ipynb\", line 24, in <module>\n    },\n  File \"/opt/conda/lib/python3.6/site-packages/pandas/util/_decorators.py\", line 118, in wrapper\n    return func(*args, **kwargs)\n  File \"/opt/conda/lib/python3.6/site-packages/pandas/io/excel.py\", line 230, in read_excel\n    io = ExcelFile(io, engine=engine)\n  File \"/opt/conda/lib/python3.6/site-packages/pandas/io/excel.py\", line 263, in __init__\n    raise ImportError(err_msg)\nImportError: Install xlrd >= 0.9.0 for Excel support\n"}]}